packages/components/src/styles/ProgressBar.module.css

```
@@ -9,16 +9,52 @@
 }
 
 .progress {
-	--fill: var(--lp-color-gray-500);
+}
+
+.spinner {
+	--fill-color: var(--lp-color-gray-500);
 	display: inline-flex;
 }
 
+.bar {
+	--fill-color: var(--lp-color-brand-cyan-base);
+	display: flex;
+	flex-direction: column;
+	gap: var(--lp-spacing-300);
+}
+
 .outerCircle {
 	stroke: var(--lp-color-white-100);
 }
 
 .innerCircle {
-	stroke: var(--fill);
+	stroke: var(--fill-color);
+}
+
+.container {
+	display: flex;
+}
+
+.label {
+	flex: 1;
+	font: var(--lp-text-label-1-semibold);
```

If we do need this, then I'd suggest we align this with our other labels:
```suggestion
	font: var(--lp-text-label-1-medium);
```